<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Job Editor</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>560</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>1. Provide your URL and click on Fetch Page. This will get all links non-recursively</string>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,10,1">
    <item row="0" column="1">
     <widget class="QComboBox" name="cmbPageUrl">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>URL:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnFetchPage">
      <property name="text">
       <string>Fetch Page</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>601</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>2. Select files to get</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="4" colspan="2">
     <widget class="QTreeWidget" name="treeFileSelector">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnCheckAllShown">
      <property name="text">
       <string>Check all shown</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Filter:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtSelectionFilter"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnUncheckAllShown">
      <property name="text">
       <string>Uncheck all shown</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnResetSelection">
      <property name="text">
       <string>Start all over</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>4. Set target local folder</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,10,1">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Download into: </string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnBrowseLocalTarget">
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cmbLocalTarget">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>461</width>
     <height>531</height>
    </rect>
   </property>
   <property name="title">
    <string>Preview file list</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="lstFilesetPreview">
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>601</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>5. Select post-download action</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="1" column="2">
     <widget class="QComboBox" name="cmbPostDownloadAction">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>6. Name your job</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtJobName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
